### ISSUE TITLE
Build Windows package

### ISSUE THREAD

**Issue by vsuhachev** [vldgeorgiev/therion#10]
Any idea about how to build Therion for Windows?

**Comment by vldgeorgiev** [comment]
To build the exe, you just have to adjust the makefile, if that's what you mean. There are already a couple of scripts that do that. Running "make config-win32" switches the make to a Windows build.

I will also make a new installer for Windows because I don't have the source for the old one. I just haven't seen much interest in the Therion users...

**Comment by vsuhachev** [comment]
Is win32 build should run on Windows?

Many users is not in TherionList, not understand English but need updates. For example - outdated 'ru' translations is a problem for local users...

**Comment by vldgeorgiev** [comment]
Yes, config-win32 build will produce the therion.exe, which the users will have to copy over the old installation in Program Files.
I also keep the compiled EXE here https://github.com/vldgeorgiev/therion/wiki and update it after each change, so you can reuse it.

**Comment by vldgeorgiev** [comment]
If you think there are enough users that need it, I will make the complete Windows installer. And maybe some scripts to automate it.
